.travis.yml

```diff
@@ -127,8 +127,10 @@ matrix:
 install:
     - git clone git://github.com/astropy/ci-helpers.git
     - source ci-helpers/travis/setup_conda_$TRAVIS_OS_NAME.sh
+    - pip install https://github.com/keflavich/httpbin/archive/master.zip
```

Comment:
yes, the script basically runs ``pip install $PIP_DEPENDENCIES``, so you can add anything you can install with pip